platform/src/main/java/org/stellar/anchor/platform/event/EventProcessorManager.java

```diff
@@ -246,20 +246,21 @@ void handleEvent(AnchorEvent event) {
             }
           }
         } catch (IOException ioex) {
-          // ***** Error #1: connection error
-          getHttpErrorBackoffTimer().reset();
-          httpStatusNotOkAttempts = 0;
-          try {
-            getNetworkBackoffTimer().backoff();
-          } catch (InterruptedException e) {
-            // The thread is interrupted, so we need to stop the processor. This will
-            // break the while loop.
-            currentThread().interrupt();
+          // Retry for connection error
+          if (++networkErrorRetryAttempts <= NETWORK_MAX_RETRIES) {
+            try {
+              getNetworkBackoffTimer().backoff();
+            } catch (InterruptedException e) {
+              // The thread is interrupted, so we need to stop the processor. This will
+              // break the while loop.
+              currentThread().interrupt();
+            }
```

Comment:
Why are we catching an `InterruptedException` here for the network backoff timer but not for the http status backoff timer?

Reply:
Will catch for HttpErrorBackoffTimer as well